profile.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
```

Comment:
There is no OOPS concept applied here.
Make sure that all files are class based code and oops concepts are used properly.